src/js/clickable-component.js

```diff
@@ -110,8 +127,8 @@ class ClickableComponent extends Component {
   /**
    * Allows sub components to stack CSS class names
    *
-   * @return {String}
-   * @method buildCSSClass
+   * @return {string}
+   *         The current css class for this ClickableComponent
```

Comment:
This doesn't feel particularly accurate. It returns the default `className` for the component's element - not the current `className`. I think we should avoid using the term "css class" because it's got nothing to do with CSS.